internal/serve/serve.go

```diff
@@ -4,22 +4,31 @@ import (
 	"fmt"
 	"net/http"
 
+	"wallet-backend/internal/data"
+	"wallet-backend/internal/db"
+
+	"github.com/pkg/errors"
 	supporthttp "github.com/stellar/go/support/http"
 	supportlog "github.com/stellar/go/support/log"
 	"github.com/stellar/go/support/render/health"
 )
 
 type Configs struct {
-	Logger *supportlog.Entry
-	Port   int
+	Logger       *supportlog.Entry
+	Port         int
+	DBConnection string
```

Comment:
❓ what is it going to look like? A database URL?

Reply:
Yes, I've renamed it: [b8f9b6c](https://github.com/stellar/wallet-backend/pull/5/commits/b8f9b6c74e681f3efa3a5be00689cb9eea27db12)